### DISCUSSION TITLE
Can I update the version of some MIUI apps in this module on APK mirror?

### DISCUSSION THREAD

**Discussion by KolbyReiuji** [reiryuki/Miui-Core-Magisk-Module#10] [Q&A]
Hi. So I decide to update some MIUI apps in this module by grab the latest version on APK mirror, is it possible to update them?(eg: MIUI system launcher)

**Comment by joeyoropesa-dev** [comment]
Dev is probably ignoring us so that we need to learn on our own (make mistakes with our devices and one of our actions should success - so basically risking our devices for knowledge)


That's how I need to experiment with porting Miui System from an Global ROM to the module and to be sure that it will be compatible with every device model


There is also many discussions that was made and never gets an response but that's life...


When you're not higher or in the same level like the dev you want an attention, you will never get an attention so - no attentions/responses, no answers and no successful bugs reporting


Especially in 2023+


Sorry.

**Comment by joeyoropesa-dev** [comment]
But by me to answer your question, if the for example MiuiCore is based on MIUI 13, you cannot download latest MIUI apps that requires MIUI 14 to work - if you do that, welcome to the bootloop



Also, most apps you want to update requires permissions update inside the system/etc/permissions dir and some probably extra frameworks and etc... especially if you want to update some core apps of MIUI



So to be sure, don't update - some apps are meant to be updated but only to the higher possible version for the target MIUI version, not higher than MiuiCore base version that is using


But some apps are meant to be updated via System Updates and some doesn't even gets updated for the reasons I writted above.




(If I succeed in tests and make my own more universal and newer MiuiCore, I will be more friendly to everyone and I will listen to everyone's suggestions and reports and questions no matter how big or small they are - they are all equal for me 🫂✨)

**Comment by reiryuki** [comment]
Miui Core is not for Miui/HyperOS ROM
